webapp/src/api/api.ts

```diff
@@ -0,0 +1,49 @@
+import {PlaceType} from "../types/PlaceType";
+
+export async function getPlaces() : Promise<PlaceType[]> {
+    const apiEndPoint = "http://localhost:5000/api"
+    let response = await fetch(apiEndPoint + "/places/list");
+    return response.json();
+}
+
+export async function addPlace(place: PlaceType) : Promise<boolean> {
+    const apiEndPoint = "http://localhost:5000/api"
```

Comment:
We have to take into account that may be in the future this url must be changed as the Web app would be deployed in a machine

Reply:
We will see when we deploy everything if any changes are needed.